### PR TITLE
chore_: bump waku

### PR DESCRIPTION
**Pull request by plopezlpz**
Bump go-waku to fix issues with concurrent map access

### CODE REVIEW

**Review by richard-ramos — APPROVED** [review]

**Comment by churik** [comment]
Tested this in https://github.com/status-im/status-mobile/pull/21815 and comparison with release, and results are so (tested receiving messages in 5G in 1-1 chat):


  | PR | Release
-- | -- | --
message 1 | 1 sec | 30 sec
message 2 | 80 sec| 30 sec
message 3 | 9 sec| 40 sec
message 4 | 1 sec| 45 sec
message 5 | 80 sec | 80 sec
message 6 | waited for 300 sec, haven't received anyting | waited for 300 sec, haven't received anything

After re-login can receive messages in both chats
Given that, I don't see any change.
I haven't seen crashes here, if you can reassusre that this PR won't introduce additional regression, I belive it is OK to merge this. 
Haven't seen any issues on wifi / offline - online, communities fetching, but haven't had enough time and described scope to test it thoroughly.

**Comment by ilmotta** [comment]
@churik Thank you so much for running the experiments and bringing some data to the table. In my opinion there's not enough data to make a truly informed decision. What you provided is enough for me to believe merging this PR is not going to make things worse in terms of the messaging experience, but might reduce the frequency of crashes. Because of that, I think there's a small upside and the risk to merge is acceptable.

We can also keep checking the app in the upcoming days next week until it's approved by app stores and if we find any outstanding bug we can still reevaluate if it's worth publishing the final build to users or not.

**Review by ilmotta — APPROVED** [review]

**Comment by plopezlpz** [comment]
@igor-sirotin @osmaczko can you merge?

**Pull request merged**
